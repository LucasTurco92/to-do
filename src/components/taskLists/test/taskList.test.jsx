import { render } from '@testing-library/react';
import TaskLists from '../taskLists';

it('When it invokes the TaskLists component, should render it properly', ()=>{

    const { container } = render(<TaskLists />);
    
    expect(container).toMatchSnapshot();
});
