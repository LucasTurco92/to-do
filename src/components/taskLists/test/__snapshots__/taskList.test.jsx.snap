// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When it invokes the TaskLists component, should render it properly 1`] = `
<div>
  <div
    class="sc-dkPtRN gnGpTz"
  >
    <div
      class="sc-bdvvtL iIcWDR"
    >
      <div
        class="sc-gsDKAQ bEunUZ"
        color="var(--orange)"
      >
        Pending
      </div>
    </div>
    <div
      class="sc-bdvvtL iIcWDR"
    >
      <div
        class="sc-gsDKAQ dniuBu"
        color="var(--green)"
      >
        Completed
      </div>
    </div>
  </div>
</div>
`;
